src/groundStation/groundStation.py

```diff
@@ -367,6 +391,7 @@ def getOptions(self):
     while True:
         try:
             server, port, toSend = csp.getInput(prompt='to send: ')
+            csp.handlePipeMode()
```

Comment:
We should probably also handle `handlePipeMode()` in cli.py (on line 36: `gs.handlePipeMode()`) so that this functionality is also available to the cli.

Reply:
Great catch, thx.